src/current/v25.1/online-schema-changes.md

```diff
@@ -80,6 +80,7 @@ The following statements use the declarative schema changer by default:
 - [`ALTER TABLE ... ADD CONSTRAINT ... FOREIGN KEY ... NOT VALID`]({% link {{ page.version.version }}/alter-table.md %}#add-constraint)
 - [`ALTER TABLE ... VALIDATE CONSTRAINT`]({% link {{ page.version.version }}/alter-table.md %}#drop-constraint)
 - [`ALTER TABLE ... DROP CONSTRAINT`]({% link {{ page.version.version }}/alter-table.md %}#validate-constraint)
+- [`ALTER TABLE ... ALTER COLUMN TYPE`]({% link {{ page.version.version }}/alter-table.md %}#alter-column-data-type)
```

Comment:
This looks good, but is it meant to be an exhaustive list? I think a few operations are missing (e.g., DROP INDEX, CREATE/DROP TRIGGER, DROP VIEW, COMMENT ON, etc.). I often refer to this as a list of supported operations in the DSC: https://docs.google.com/spreadsheets/d/1eVjaSpZHDgh6xZlp-GN2w7ELK2Iy5omqL36LJjJMJd8/edit?gid=0#gid=0

Reply:
thanks @spilchen - i've filed https://cockroachlabs.atlassian.net/browse/DOC-12360 to do the work of updating that list

(AIUI it is meant to be exhaustive)